pkgs/development/ocaml-modules/irmin/default.nix

```diff
@@ -27,5 +22,4 @@ buildDunePackage rec {
     license = lib.licenses.isc;
     maintainers = [ lib.maintainers.vbgl ];
   };
-
-}
+} // (import ./src.nix { inherit fetchurl; }))
```

Comment:
Why don’t you inherit the `src` and `version` attributes from `ppx_irmin`? I find this clearer but maybe it is just a matter of taste.

Reply:
I agree that's cleaner, as we don't have to duplicate the `meta` set. Thanks!